tests/ssg_test_suite/rule.py

```diff
@@ -39,7 +39,7 @@ def get_viable_profiles(selected_profiles, datastream, benchmark, script=None):
     all_profiles.append(OSCAP_PROFILE_ALL_ID)
 
     for ds_profile in all_profiles:
-        if 'ALL' in selected_profiles:
+        if 'ALL' in selected_profiles or OSCAP_PROFILE_ALL_ID in selected_profiles:
```

Comment:
Just bringing this up because I hit it again: there's no 1.2.18 release of OpenSCAP with support for `(all)` which makes this effectively broken on systems without that patch backported (Debian, all Ubuntu...).